pvm/ts/tests/bugs.test.ts

```
@@ -239,5 +240,18 @@ describe("Texas Holdem - Data driven", () => {
             expect(legalActions[2].action).toEqual("raise");
             expect(legalActions[2].min).toEqual("60000000000000000");
         });
+
+        it.only("should test bug 984 second test", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)